src/flight_paths.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from graph import Graph
+from distance import calculate_distance
+
+
+def import_cities():
+    '''
+        Import the city flight connection path JSON file,
+        Returns dictionary of cities
+    '''
+    # url = '///cities_with_airports.json'
+    url = 'https://codefellows.github.io/sea-python-401d4/_downloads/cities_with_airports.json'
+    r = requests.get(url)
+    cities = r.json()   # converts json to dictionary
```

Comment:
Noice. instead of using `requests` to pull it from the class website though you could've just downloaded the data file yourself and used `json.loads`. Either way, it works.